altinity-ubuntu2204-libvirt.json

```diff
@@ -0,0 +1,131 @@
+ {
+  "variables": {
+    "box_version": "{{env `VERSION`}}"
+  },
+  "provisioners": [
+    {
+      "scripts": [
+        "scripts/altinity/ubuntu2204/apt.sh",
+        "scripts/altinity/ubuntu2204/floppy.sh",
+        "scripts/altinity/ubuntu2204/fixkvp.sh",
+        "scripts/altinity/ubuntu2204/network.sh"
+      ],
+      "type": "shell",
+      "timeout": "120m",
+      "start_retry_timeout": "15m",
+      "expect_disconnect": "true",
+      "only": [
+        "altinity-ubuntu2204-libvirt"
+      ]
+    },
+    {
+      "scripts": [
+        "scripts/altinity/ubuntu2204/vagrant.sh",
+        "scripts/altinity/ubuntu2204/profile.sh",
+        "scripts/altinity/ubuntu2204/motd.sh",
+        "scripts/altinity/ubuntu2204/fixtty.sh",
+        "scripts/altinity/ubuntu2204/virtualbox.sh",
+        "scripts/altinity/ubuntu2204/parallels.sh",
+        "scripts/altinity/ubuntu2204/vmware.sh",
+        "scripts/altinity/ubuntu2204/qemu.sh",
+        "scripts/altinity/ubuntu2204/cleanup.sh",
+        "scripts/altinity/ubuntu2204/ubuntu_ami_for_ci.sh"
+      ],
+      "type": "shell",
+      "timeout": "120m",
+      "pause_before": "120s",
+      "start_retry_timeout": "15m",
+      "expect_disconnect": "true",
+      "only": [
+        "altinity-ubuntu2204-libvirt"
+      ]
+    },
+    {
+      "scripts": [
+        "scripts/common/motd.sh",
+        "scripts/common/keys.sh",
+        "scripts/common/machine.sh",
+        "scripts/common/leases.sh",
+        "scripts/common/localtime.sh",
+        "scripts/common/zerodisk.sh",
+        "scripts/common/lockout.sh"
+      ],
+      "type": "shell",
+      "timeout": "120m",
+      "start_retry_timeout": "15m",
+      "expect_disconnect": "true"
+    }
+  ],
+  "builders": [
+    {
+      "type": "qemu",
+      "name": "altinity-ubuntu2204-libvirt",
+      "vm_name": "altinity-ubuntu2204-libvirt",
+      "output_directory": "output/altinity-ubuntu2204-libvirt",
+      "qemu_binary": "qemu-system-x86_64",
+      "boot_wait": "10s",
```

Comment:
Maybe increase this to "15s" or "20s" to address hanging builds?

Reply:
Updated :+1: